--- docbook-xsl-1.67.2/common/titles.xsl.orig	2004-11-16 19:01:36.000000000 +0100
+++ docbook-xsl-1.67.2/common/titles.xsl	2004-12-20 23:09:58.741306392 +0100
@@ -653,12 +653,6 @@
     <xsl:otherwise>
       <xsl:apply-templates select="$target" mode="xref-to-prefix"/>
 
-      <xsl:if test="$target/title or $target/*/title">
-	<xsl:attribute name="title">
-	  <xsl:apply-templates select="$target" mode="xref-title"/>
-	</xsl:attribute>
-      </xsl:if>
-
       <xsl:apply-templates select="$target" mode="xref-to">
 	<xsl:with-param name="referrer" select="."/>
 	<xsl:with-param name="xrefstyle">
